test/config.test1.yaml

```diff
@@ -0,0 +1,337 @@
+
+logging:
+  level: INFO
+  format: "%(levelname)s:%(name)s:%(message)s"
+
+scenario:
+  simpl: ['']
+  ll: ['copt']
+  clusters: [5]
+  opts: [Co2L-24H]
+
+countries: ["NG", "BJ"]
+
+  # ['DZ', 'AO', 'BJ', 'BW', 'BF', 'BI', 'CM', 'CF', 'TD', 'CG', 'CD',
+  # 'DJ', 'EG', 'GQ', 'ER', 'ET', 'GA', 'GH', 'GN', 'CI', 'KE', 'LS', 'LR', 'LY',
+  # 'MG', 'MW', 'ML', 'MR', 'MU', 'MA', 'MZ', 'NA', 'NE', 'NG', 'RW',
+  # 'SL', 'ZA', 'SS', 'SD', 'SZ', 'TZ', 'TG', 'TN', 'UG', 'ZM', 'ZW']  # list(AFRICA_CC)
+
+  #["NG"]  # Nigeria
+  #["NE"]  # Niger
+  #["SL"]  # Sierra Leone
+  #["MA"]  # Morroco
+  #["ZA"]  # South Africa
+
+# if True clusters to GADM shapes, if False Voronoi cells will be clustered
+alternative_clustering: False
+
+# options for build_shapes
+build_shape_options:
+    gadm_layer_id: 1  # GADM level area used for the gadm_shapes. Codes are country-dependent but roughly: 0: country, 1: region/county-like, 2: municipality-like
+    update_file: False  # When true, all the input files are downloaded again and replace the existing files
+    out_logging: True  # When true, logging is printed to console
+    year: 2020  # reference year used to derive shapes, info on population and info on GDP
+    nprocesses: 5  # number of processes to be used in build_shapes
+
+osm_data_cleaning_options:
+  names_by_shapes: True  # Set the country name based on the extended country shapes
+  threshold_voltage: 35000  # [V] minimum voltage threshold to keep the asset (cable, line, generator, etc.) [V]
+  tag_substation: "transmission"  # needed feature tag to be considered for the analysis. If empty, no filtering on the tag_substation is performed
+  add_line_endings: True  # When true, the line endings are added to the dataset of the substations
+  group_close_buses: True  # When true, the dataset of buses is filtered to merge close nodes and guarantee the matching among line endings in the same bus by voltage
+
+base_network:
+  min_voltage_substation_offshore: 35000  # [V] minimum voltage of the offshore substations
+  min_voltage_rebase_voltage: 35000
+
+load_options:
+  weather_year: 2014
+  prediction_year: 2030
+  scale: 1
+
+summary_dir: results
+
+snapshots:
+  start: "2013-03-1"
+  end: "2013-03-8"
+  closed: "left" # end is not inclusive
+
+enable:
+  # prepare_links_p_nom: false
+  retrieve_databundle: false
+  download_osm_data: true
+  # retrieve_cutout: true
+  # retrieve_natura_raster: true
+  # If "build_cutout" : true # requires cds API key https://cds.climate.copernicus.eu/api-how-to
+  # More information https://atlite.readthedocs.io/en/latest/introduction.html#datasets
+  build_cutout: false
+  build_natura_raster: true  # If True, then build_natura_raster can be run
+  # custom_busmap: false
+
+electricity:
+  voltages: [220., 300., 380.]
+  co2limit: 7.75e+7 # 0.05 * 3.1e9*0.5
+  co2base: 1.487e9
+  agg_p_nom_limits: data/agg_p_nom_minmax.csv
+
+  extendable_carriers:
+    Generator: []
+    StorageUnit: [] # battery, H2
```

Comment:
maybe worth checking the other features like generator, link etc. at a later stage